File: Java/Aksesoris.java
class Aksesoris extends PetShop {
    private String jenis;
    private String bahan;
    private String warna;

    public Aksesoris() {
        this.jenis = "";
        this.bahan = "";
        this.warna = "";
    }

    public void setJenis(String jenis) {
        this.jenis = jenis;
    }

    public void setBahan(String bahan) {
        this.bahan = bahan;
    }

    public void setWarna(String warna) {
        this.warna = warna;
    }

    public String getJenis() {
        return jenis;
    }

    public String getBahan() {
        return bahan;
    }

    public String getWarna() {
        return warna;
    }
}

File: CPP/Aksesoris.cpp
#include <iostream>
#include <string>
#include <vector>
#include "Petshop.cpp"
using namespace std;

class Aksesoris : public PetShop {
    private:
        string jenis, bahan, warna;
    
    public:
        // Konstruktor kosongan
        Aksesoris() : jenis(""), bahan(""), warna("") {}
    
        // Destruktor
        ~Aksesoris() {
    
        }
    
        void set_jenis(string jenis) { this->jenis = jenis; }
        void set_bahan(string bahan) { this->bahan = bahan; }
        void set_warna(string warna) { this->warna = warna; }
    
        string get_jenis() const { return jenis; }
        string get_bahan() const { return bahan; }
        string get_warna() const { return warna; }
    };
    